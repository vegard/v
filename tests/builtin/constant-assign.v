constant { x := u64 123 };
x = u64 456;
print x;
