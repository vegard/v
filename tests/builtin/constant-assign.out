tests/builtin/constant-assign.v:2:0: cannot assign to constant
x = u64 456;
^^^^^^^^^^^
